.github/workflows/release.yml

```diff
@@ -216,6 +155,50 @@ jobs:
           if [ "${{ inputs.dry_run }}" = "true" ]; then  
             box_summary="[Dry run] "
           fi
+       # BOX_FOLDER_ID is an env var that is set inside the promote-release-to-box.py script file
+          # echo "$box_summary :ballot_box: ${{ inputs.release_version }} was uploaded to https://pentaho.app.box.com/folder/$BOX_FOLDER_ID" >> $GITHUB_STEP_SUMMARY
```

Comment:
This echo must be uncommented. It's relevant information to show in the summary

Reply:
uncommented